packages/AutoTDD-Tests.package/ATDDGitHubActionParserTest.class/README.md

```diff
@@ -0,0 +1 @@
+An ATDDGitHubActionParserTest is class that tests the methods from the ATDDGitHubActionsParser whether they parse as they should to. 
```

Comment:
```suggestion
Remove class comment
```
There are currently no class comments for test classes, let's keep it this way so we don't make ourselves responsible for even more :D

Reply:
I will be glad to remove it hah